src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddFileMetadataToDeposit.scala

```diff
@@ -17,32 +17,103 @@ package nl.knaw.dans.easy.multideposit.actions
 
   val (datasetID, dataset) = entry
 
-  // TODO @rvanheest move the mimetype computation to a separate action once the new
-  // parameterized version of Action is installed?
-  lazy val mimetypeMap: Try[List[(File, MimeType)]] = Try {
+  case class FileInstruction(file: File, title: Option[String], subtitles: Seq[Subtitle])
+
+  sealed abstract class AudioVideo(val vocabulary: String)
+  case object Audio extends AudioVideo("http://schema.org/AudioObject")
+  case object Video extends AudioVideo("http://schema.org/VideoObject")
+
+  case class Subtitle(filepath: File, language: Option[String])
+
+  class FileMetadata(val filepath: File, val mimeType: MimeType)
+  case class AVFileMetadata(override val filepath: File,
+                            override val mimeType: MimeType,
+                            vocabulary: AudioVideo,
+                            title: String,
+                            accessibleTo: FileAccessRights.UserCategory,
+                            subtitles: Seq[Subtitle]) extends FileMetadata(filepath, mimeType)
+
+  private lazy val fileInstructions: Map[File, FileInstruction] = {
+    def resolve(filepath: String): File = new File(settings.multidepositDir, filepath).getAbsoluteFile
+
+    dataset.getColumns("AV_FILE", "AV_FILE_TITLE", "AV_SUBTITLES", "AV_SUBTITLES_LANGUAGE")
+      .toRows
+      .map(row => (
+        row.get("AV_FILE"),
+        row.get("AV_FILE_TITLE"),
+        row.get("AV_SUBTITLES"),
+        row.get("AV_SUBTITLES_LANGUAGE")
+      ))
+      .collect {
+        case (maybeFile, maybeTitle, maybeSub, maybeSubLang) if maybeFile.exists(!_.isBlank) =>
+          (resolve(maybeFile.get), maybeTitle, maybeSub.map(resolve), maybeSubLang)
+      }
+      .groupBy { case (file, _, _, _) => file }
+      .map { case (file, instr) =>
+        val optTitle = instr.collectFirst { case (_, Some(title), _, _) => title }
+        val subtitles = instr.collect { case (_, _, Some(sub), maybeSubLang) => Subtitle(sub, maybeSubLang) }
+        file -> FileInstruction(file, optTitle, subtitles)
+      }
+  }
+
+  private lazy val avAccessibility: FileAccessRights.Value = {
+    dataset.findValue("SF_ACCESSIBILITY")
+      .flatMap(FileAccessRights.valueOf)
+      .orElse(dataset.findValue("DDM_ACCESSRIGHTS")
+        .map(v => FileAccessRights.accessibleTo(AccessCategory.valueOf(v))))
+      .getOrElse(throw ActionException(row, "No value found for either SF_ACCESSIBILITY or DDM_ACCESSRIGHTS"))
+  }
+
+  private def getFileMetadata(file: File): Try[FileMetadata] = {
+    def mkFileMetadata(m: MimeType, voca: AudioVideo): AVFileMetadata = {
+      fileInstructions.get(file) match {
+        case Some(FileInstruction(_, optTitle, subtitles)) =>
+          val title = optTitle.getOrElse(file.getName)
+          AVFileMetadata(file, m, voca, title, avAccessibility, subtitles)
+        case None =>
+          AVFileMetadata(file, m, voca, file.getName, avAccessibility, Seq.empty)
```

Comment:
I guess, given the issue description:

> neem dcterms:title op met waarde AV_FILE_TITLE, default naar de bestandsnaam

So if there are no explicit file instructions for a particular file, you just use the filename itself